.konchrc

```diff
@@ -1,22 +1,19 @@
+#!/opt/conda/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
The encoding pragma is unnecessary as of Python 3, and konch no longer supports Python 2. This should be removed.